pcsx2/gui/FrameForGS.cpp

```diff
@@ -642,21 +641,19 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 	out << std::fixed << std::setprecision(2) << fps;
 	OSDmonitor(Color_StrongGreen, "FPS:", out.str());
 
-	int new_width = 0, new_height = 0;
+	const u64& smode2 = *(u64*)PS2GS_BASE(GS_SMODE2);
+	int new_width = 0, new_height = 0, interlaced = !g_Conf->GSWindow.DisableScalingInterlaceCompensation && (smode2 & 2) && (smode2 & 2);
```

Comment:
I think ``interlaced`` variable should be a bool.

```
(smode2 & 2) && (smode2 & 2)
```

I guess this is some sort of mistake? I'd prefer if you could just get the INT/FFMD bit-field instead of doing logical operations on the whole register, definitely helps in the aspect of code readability.

Reply:
Yes, it could be a bool. I used it differently at first and forgot to change it. The mistake is likely a result of the same change in use.

The smode2 code was already there. I assumed it was the easiest way to get the INT/FFMD data, defenitely the easiest way for me. 😛

Reply:
> The smode2 code was already there. I assumed it was the easiest way to get the INT/FFMD data, definitely the easiest way for me.

Easy for you, but not everyone knows the position of INT and FFMD fields in SMODE2. 😛 

Bonus points for also changing the older code into proper bitfield name of the register. I hate codes like that without any proper comments and doing bitwise operations on the whole register, have to look at the documents all the time to know which field they're messing up with. -_-